src/js/personalization/user-profile-beta/actions/index.js

```diff
@@ -53,7 +53,8 @@ async function sendProfileRequests() {
     ['email', '/profile/email'],
     ['primaryTelephone', '/profile/primary_phone'],
     ['alternateTelephone', '/profile/alternate_phone'],
-    ['mailingAddress', '/profile/mailing_address']
+    ['mailingAddress', '/profile/mailing_address'],
+    ['serviceHistory', '/profile/service_history']
```

Comment:
The main point here is that anytime you had a new element to that array, it will become part of the profile object. The first element of each array is the property name, and the second is the corresponding API route.